core/staking_verifier_test.go

```diff
@@ -1000,9 +1002,42 @@ func TestVerifyAndDelegateFromMsg(t *testing.T) {
 			}(),
 			expAmt: tenKOnes,
 		},
+		{
+			// 17: small amount v2
+			sdb: makeStateDBForStake(t),
+			msg: func() staking.Delegate {
+				msg := defaultMsgDelegate()
+				msg.Amount = new(big.Int).Mul(big.NewInt(90), oneBig)
+				return msg
+			}(),
+			ds:         makeMsgCollectRewards(),
+			epoch:      big.NewInt(100),
```

Comment:
Is there any special meaning of setting `epoch` to 100 here? Or is it just consistent with the min delegation value of 100

Reply:
It's just a random number which is used to differentiate the epoch before and after the fork (in tests)